ci/tasks/deploy_gcp.yaml

```diff
@@ -0,0 +1,26 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+    tag: alpine
+
+params:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+
+inputs:
+- name: ons-design-system-release-artifact
+
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      export GOOGLE_APPLICATION_CREDENTIALS=~/gcloud-service-key.json
+      cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+      
+      gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+
+      gsutil -m cp -r ons-design-system-release-artifact/** gs://census-eq-cdn/design-system/
```

Comment:
Is the bucket name being introduced as an env var?

Reply:
Yes.